docs/ops/arithmetic/Sqrt_1.md

```diff
@@ -25,13 +23,30 @@
 *Sqrt* does the following with the input tensor *a*:
 
 \f[
-a_{i} = sqrt(a_{i})
+a_{i} = \sqrt{a_{i}}
```

Comment:
Before: 
![image](https://user-images.githubusercontent.com/20084299/115863943-7b5fd780-a436-11eb-86ed-07f474144d4d.png)
after:
![image](https://user-images.githubusercontent.com/20084299/115863905-6d11bb80-a436-11eb-9cee-4cdc91718d14.png)

Reply:
I would express this formula differently:
`Sqrt{a} = sqrt(a)`
because we already mentioned the op is element-wise, or
`o_{i} = sqrt(a_{i})`
But the idea here is avoid using the same symbol for both sides in the formula

Reply:
Make sense, changed to `o_{i} = \sqrt(a_{i})` where `o` is the output tensor